client/landing/stepper/declarative-flow/start-writing.ts

```diff
@@ -108,7 +108,7 @@ const startWriting: Flow = {
 				case 'plans':
 					if ( siteSlug ) {
 						await updateLaunchpadSettings( siteSlug, {
-							checklist_statuses: { plan_completed: true },
+							checklist_statuses: { plan_completed: true, domain_upsell_deferred: true },
```

Comment:
Since paid domains don't submit, we need to inform we achieved `domain_upsell_deferred` here.

Reply:
Yeah, that would make sense, thanks for the suggestion! I ended up [simplifying the implementation in this commit](https://github.com/Automattic/wp-calypso/pull/76634/commits/acced02a00bf17abfae1ee24e1e005b578c60502).